itests/util/src/main/java/org/apache/hadoop/hive/cli/control/CoreCliDriver.java

```diff
@@ -53,15 +53,14 @@ public void beforeClass() {
     final String hiveConfDir = cliConfig.getHiveConfDir();
     final String initScript = cliConfig.getInitScript();
     final String cleanupScript = cliConfig.getCleanupScript();
-    final boolean useHBaseMetastore = cliConfig.getMetastoreType() == MetastoreType.hbase;
     try {
       final String hadoopVer = cliConfig.getHadoopVersion();
 
       qt = new ElapsedTimeLoggingWrapper<QTestUtil>() {
         @Override
         public QTestUtil invokeInternal() throws Exception {
           return new QTestUtil((cliConfig.getResultsDir()), (cliConfig.getLogDir()), miniMR,
-              hiveConfDir, hadoopVer, initScript, cleanupScript, useHBaseMetastore, true,
+              hiveConfDir, hadoopVer, initScript, cleanupScript, true,
               cliConfig.getFsType());
```

Comment:
In the original PR this line is moved up (remove previous line break = 63 and 64 are merged)